charts/k8s-monitoring/docs/examples/platforms/openshift/README.md

```diff
@@ -2,7 +2,19 @@
 (NOTE: Do not edit README.md directly. It is a generated file!)
 (      To make changes, please modify values.yaml or description.txt and run `make examples`)
 -->
-# Example: platforms/openshift/values.yaml
+# OpenShift
+
+This example shows the modifications requires to deploy to an OpenShift cluster.
+
+These modifications skip the deployment of kube-state-metrics and Node Exporter, since they will already be present on
+the cluster, and adjust the configuration to Grafana Alloy to find those existing components.
+It also modifies Grafana Alloy to lock down security and permissions.
+
+The `platform: openshift` switch also creates SecurityContextConstraints objects that modifies the permissions for the
+Grafana Alloy.
+
+Note that these alloy pods cannot enable `readOnlyRootFilesystem` because they require being able to write to their
+storage path, which defaults to `/tmp/alloy`.
```

Comment:
```suggestion
This example shows the modifications requires to deploy to an OpenShift cluster.

These modifications:
- Skip the deployment of kube-state-metrics and Node Exporter, since they are already present on
the cluster
- Adjust the configuration to Grafana Alloy to find those existing components
- Make changes to Grafana Alloy to lock down security and permissions

The `platform: openshift` switch also creates SecurityContextConstraints objects that modifiy the permissions for the
Grafana Alloy.

Note that these Alloy pods cannot enable `readOnlyRootFilesystem` because they require being able to write to their
storage path, which defaults to `/tmp/alloy`.
```